README.md

```diff
@@ -131,6 +136,25 @@ Then follow the steps to [run Atoti UI](https://activeviam.com/activeui/document
 
 Finally open your dashboards, widgets, filters, ... Make sure they work as expected.
 
+## Migrate an Atoti jupyter notebook
+
+To migrate a jupyter notebook, two files are necessary:
+
+- _notebook.ipynb_, the notebook to migrate. It is really important that the notebook has the `.ipynb` extension.
+- _servers.json_, the JSON file containing the server information.
+
+To get the servers information, follow the [download the servers information section](#download-the-servers-information) instructions.
+
+Now that your migration folder contains all the required file, you can run the following CLI:
```

Comment:
_"Command"_ is more correct than CLI, because CLI is for _Command line interface_. It's an interface that can contain multiple commands (like a REST interface can contain multiple endpoints).

```suggestion
Now that your migration folder contains all the required files, you can run the following command:
```